TerminalDocs/customize-settings/interaction.md

```diff
@@ -248,3 +248,19 @@ When this is set to `true`, trying to paste text with multiple lines will displa
 **Accepts:** `true`, `false`
 
 **Default value:** `true`
+
+<br />
+
+___
+
+## Legacy input encoding
+
+Force the terminal to use the legacy input encoding. Certain keys in some applications may stop working when enabling this setting. This can be useful for debugging certain types of input issues, especially with the [debug tap](https://github.com/microsoft/terminal/wiki/Enabling-the-debug-tap).
```

Comment:
Force the terminal to use the legacy input encoding. Specific keys in some applications may stop working when enabling this setting, but it can be useful for advanced-level scenarios when debugging input issues, especially with the [debug tap](https://github.com/microsoft/terminal/wiki/Enabling-the-debug-tap).